### PR TITLE
Add traefik healthcheck in docker-compose

### PR DESCRIPTION
**Pull request by trottomv**

### CODE REVIEW

**Review by pauloxnet — APPROVED** [review]

File: {{cookiecutter.project_dirname}}/traefik/conf/static.yaml [hunk 1]
```diff
@@ -13,6 +13,8 @@ entryPoints:
 log:
   level: ERROR
 
+ping: true
+
```
Comment (filippo-20tab) [inline {{cookiecutter.project_dirname}}/traefik/conf/static.yaml:17]: Is this a mistake?

File: {{cookiecutter.project_dirname}}/docker-compose.yaml [hunk 2]
```diff
@@ -46,17 +46,22 @@ services:
       - ./{{ cookiecutter.frontend_service_slug }}:/app{% endif %}
 
   traefik:
-    image: traefik:v2.6
+    command:
+      - "--configFile=/traefik/conf/static.yaml"
```
Comment (filippo-20tab) [inline {{cookiecutter.project_dirname}}/docker-compose.yaml:50]: As you're refactoring it, I'd make it more compact
```suggestion
    command: "--configFile=/traefik/conf/static.yaml"
```

File: {{cookiecutter.project_dirname}}/docker-compose.yaml [hunk 3]
```diff
@@ -46,17 +46,22 @@ services:
...
       - {{ cookiecutter.frontend_service_slug }}{% endif %}
-    volumes:
-      - ./traefik/:/traefik/:ro
-    command:
-      - "--configFile=/traefik/conf/static.yaml"
+    healthcheck:
+      test: ["CMD", "traefik" ,"healthcheck", "--ping"]
+      interval: 10s
+      timeout: 10s
+      retries: 5
```
Comment (filippo-20tab) [inline {{cookiecutter.project_dirname}}/docker-compose.yaml:58]: Why these values specifically?

**Comment by daniele20tab** [comment]
I don't understand the utility on have heathcheck on proxy services

**Review by daniele20tab — CHANGES_REQUESTED** [review]
I don't understand the utility on have heathcheck on proxy services.
And i don't see that implementation on the sprint plan.

File: {{cookiecutter.project_dirname}}/traefik/conf/static.yaml (hunk 1 above)
Comment (trottomv) [reply {{cookiecutter.project_dirname}}/traefik/conf/static.yaml:17]: not a mistake, this would enable ping for healthcheck

**Comment by trottomv** [comment]
> I don't understand the utility on have heathcheck on proxy services. And i don't see that implementation on the sprint plan.

we can consider it a spike to evaluate the effective need and close the PR without problems

File: {{cookiecutter.project_dirname}}/docker-compose.yaml (hunk 3 above)
Comment (trottomv) [reply {{cookiecutter.project_dirname}}/docker-compose.yaml:58]: these are experimental values

Comment (filippo-20tab) [reply {{cookiecutter.project_dirname}}/docker-compose.yaml:58]: Until a solid reason arise to use different values, I suggest adopting the default ones. The main Talos branch should not integrate experimental features.

**Comment by trottomv** [comment]
to assess the real need in the future

**Pull request closed without merging**